source/AE8_AP8.f

```diff
@@ -51,7 +51,7 @@
 !	whatf -> which kind of flux, 1=differential 2=E range 3=integral (long integer)
 !	Nene -> Number of energy channels to compute
 !	energy -> energy (MeV) at which fluxes must be computed (double array [2,25])
-!	iyear,idoy,UT -> times when flux are to be computed (not usefull if imput position is not in GSE, GSM, SM,GEI) (respectively long array(ntime_max), long array(ntime_max), double array(ntime_max))
+!	iyear,idoy,UT -> times when flux are to be computed (not usefull if imput position is not in GSE, GSM, SM,GEI) (respectively long array(ntime), long array(ntime), double array(ntime))
```

Comment:
Since we're updating this line. "usefull"->"useful" and "imput"->"input"

Reply:
Sure, done